### PR TITLE
Update protobufs

### PR DESCRIPTION
**Pull request by xPaw**
1. Skipped Underlords protos because it requires some generation changes.
2. Removed `EnumeratePublishedFiles` because all the protos for it are gone; can't obsolete it because `CMsgCREEnumeratePublishedFilesResponse` was removed and it won't compile.
3. Combined `SteamKit2.Unified.Internal` to `SteamKit2.Internal` because some protos depend on each other and they're in separate namespaces.
    If this change is not acceptable, then protobuf generation needs to be upgraded to allow adding usings.

### CODE REVIEW

**Comment by yaakov-h** [comment]
I think this makes sense, but wouldn't mind a second opinion.

What changed need to be made for Underlords?

File: Resources/ProtobufGen/ProtobufGen/ProtobufGen.csproj [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RollForward>LatestMajor</RollForward>
```
Comment (JustArchi) [inline Resources/ProtobufGen/ProtobufGen/ProtobufGen.csproj:6]: Isn't this the default option?

Comment (yaakov-h) [reply Resources/ProtobufGen/ProtobufGen/ProtobufGen.csproj:6]: Nope. I believe the default is `LatestMinor`.

Comment (JustArchi) [reply Resources/ProtobufGen/ProtobufGen/ProtobufGen.csproj:6]: https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#runtimesdk

So freakin' complicated, apparently `Minor` is the default, thanks 😁

Comment (yaakov-h) [reply Resources/ProtobufGen/ProtobufGen/ProtobufGen.csproj:6]: Ah yeah that does make more sense...

**Comment by xPaw** [comment]
> What changes need to be made for Underlords?

Needs to bring in a bunch of more proto files like user messages, can't be bothered to deal with it.

**Comment by VoiDeD** [comment]
> I think this makes sense, but wouldn't mind a second opinion.

Most of thisl looks good to me as well - I think I originally suggested merging the internal namespaces on IRC. They're internal anywhere so the implication is that breakage here from time to time is expected.

I could go either way on `EnumeratePublishedFiles` - should it be an Obsolete stub that does a no-op, should it throw an exception, or should it be removed entirely? All three approaches seem fine.

**Comment by yaakov-h** [comment]
An obsolete stub could just cause consumers to leak resources, waiting for an event that never fires.

Throwing an exception sounds good, but removing the method will also cause an exception to be thrown... just far more cryptically.

I think all-up I'd rather keep the method, make it throw an exception, mark as obsolete and optionally hide it with `EditorBrowsableAttribute`.

**Comment by xPaw** [comment]
Anything blocking this? There are proto updates that I'd like to use.

**Pull request merged**